aspnetcore/log-mon/metrics/metrics.md

```diff
@@ -0,0 +1,174 @@
+---
+title: ASP.NET Core metrics
+description: Metrics for ASP.NET Core apps
+author: rick-anderson
+ms.author: riande
+monikerRange: '>= aspnetcore-8.0'
+ms.date: 6/30/2023
+ms.topic: article
+ms.prod: aspnet-core
+uid: log-mon/metrics/metrics
+---
+
+# ASP.NET Core metrics
+
+Metrics are numerical measurements reported over time. They're typically used to monitor the health of an app and generate alerts. For example, a web service might track how many:
+
+* Requests it received per second.
+* Milliseconds it took to respond.
+* Responses sent an error.
+
+These metrics can be reported to a monitoring system at regular intervals. If the web service is intended to respond to requests within 400 ms and starts responding in 600 ms, the monitoring system can notify the operations staff that the app response is slower than normal.
+
+## Using metrics
```

Comment:
Should mention somewhere that there are counters built into ASP.NET Core. The examples below use built-in counters but doesn't provide an explanation of where they come from.

Before .NET 8 ships we'll need a page with information about built-in counters. A lot of that content is already in GitHub issues: https://github.com/dotnet/aspnetcore/issues/47536

Also mention that an app can add its own counters using System.Diaganosts.Metrics.

Reply:
Issue for documenting built-in metrics: https://github.com/dotnet/AspNetCore.Docs/issues/29731